src/app/shared/pipes/safe.pipe.ts

```diff
@@ -106,6 +107,25 @@ export class SafePipe implements PipeTransform {
     }
   }
 
+  replaceLinksInText(textData) {
+    if (!(/<[a-z][\s\S]*>/i.test(textData))) {
+      let regex = /https?:\/\/[^\s]+/g;
+      if (typeof (textData) === 'string') {
+        let matches = textData.match(regex);
+        matches.forEach((match) => {
+          textData = textData.split(match).join(`<a href="${match}" target="_blank">${match}</a>`);
```

Comment:
Add `rel="noopener"` as well. Refer to this article for details: https://developers.google.com/web/tools/lighthouse/audits/noopener

Reply:
fixed, thanks